src/routes/docs/installation.svx

```diff
@@ -0,0 +1,16 @@
+# Installation
+
+Install using **npm**:
+```shell
+npm install -D @rgossiaux/svelte-headlessui
+```
+
+Install using **pnpm**:
+```shell
+pnpm install -D @rgossiaux/svelte-headlessui
+```
+
+Install using **yarn**:
+```shell
+yarn add @rgossiaux/svelte-headlessui
```

Comment:
Yarn command doesn't have `-D` flag while npm & pnpm do. Is this on purpose?

Reply:
Fixed it! Thanks for pointing it out. 👍️